runbooks/KubevirtHyperconvergedClusterOperatorUSModification.md

```diff
@@ -0,0 +1,32 @@
+# KubevirtHyperconvergedClusterOperatorUSModification
+
+## Meaning
+
+The HyperConverged Cluster Operator configures kubevirt and its supporting operators in an opinionated way and overwrites
+its operands when there is an unexpected change to them. Users are expected to not modify the operands directly. If 
+sucha change is a must, and is not supported by HCO API, it is possible to use special annotation to force HCO to set
+the required changes in the required operator. For more details, see
+[here](https://github.com/kubevirt/hyperconverged-cluster-operator/blob/main/docs/cluster-configuration.md#jsonpatch-annotations)
+
+However, this method may be risky and considered unsafe. 
+
+HCO fires this alert  when one of these annotations are in use.
+
+## Impact
+
+Just a warning of using unsafe modifications.
+
+
+## Diagnosis
+Check the alert details. `annotation_name` refers to specific unsafe jsonpatch annotation in the HyperConverged resource.
+
+```
+Labels
+    alertname=KubevirtHyperconvergedClusterOperatorUSModification
+    annotation_name=kubevirt.kubevirt.io/jsonpatch
+    severity=info
+```
+In the example above, the kind of specific jsonpatch annotation is `kubevirt.kubevirt.io/jsonpatch`.
+
+## Mitigation
+Not needed
```

Comment:
We are firing an alert to say that "It is unsafe." Shouldn't we recommend something here? Examples:
- Assess again whether you really need this configuration or not. If it is possible, don't use it.
- Check the HCO API again and ensure that there is no way to configure the cluster in that way with current HCO API